### ISSUE TITLE
Does an incomplete annotation rather degrade training performance??

### ISSUE THREAD

**Issue by yoonsoo-Shin** [dbolya/yolact#562]
I have 1500 chicken annotation data.
However, the annotations are not all as shown in the picture.
(The image above is one of 1500 training images.)

In this case, does it rather degrade the learning performance??
It has been training for about 12 hours and the performance is shown in the image below.

As far as I know, it is good if mAP is high, but I am as low as 10.
How can I increase the performance??

Thank you very much.

![test27](https://user-images.githubusercontent.com/58024984/97579894-f6bb7000-1a35-11eb-90c6-e5e8b3955996.png)

![캡처](https://user-images.githubusercontent.com/58024984/97579992-1783c580-1a36-11eb-9b6e-7d84ca3731e4.PNG)

**Comment by amaanda** [comment]
what exactly do you mean by incomplete annotation? you mean the objects in your image are not all labeled? I'm having a similar situation with my dataset, I was about to create an issue asking the same question... I thought about discarding the images which are incomplete but I'm not sure

**Comment by yoonsoo-Shin** [comment]
> what exactly do you mean by incomplete annotation? you mean the objects in your image are not all labeled? I'm having a similar situation with my dataset, I was about to create an issue asking the same question... I thought about discarding the images which are incomplete but I'm not sure

Yes. For example, there are 5 people in one image, but the annotation is only for 3 people.

**Comment by amaanda** [comment]
@yoonsoo-Shin it's the same situation as mine, although my object is from a different class. even though i am not completely satisfied with my results, my mAP values are higher than yours. what command are you running?

**Comment by yoonsoo-Shin** [comment]
> @yoonsoo-Shin it's the same situation as mine, although my object is from a different class. even though i am not completely satisfied with my results, my mAP values are higher than yours. what command are you running?

In the example in README.md, the command 'python eval.py --trained_model=weights/yolact_base_54_800000.pth' was used. I am using four 2080ti, so I set the batchsize to 32. 
The mAP is as low as 10, but 80% of objects are detected when a new image is inputted. I want to improve the performance further, so I am supplementing the annotation.
I'm still wondering if incomplete annotations affect performance.

**Comment by amaanda** [comment]
i see, so you're not running training but only evaluation. in my case, i am running training so i'm not sure how to help you increase your mAP.

about the incomplete annotations, we're on the same page haha i'm running a bunch of different tests trying to figure that one out.

**Comment by yoonsoo-Shin** [comment]
> i see, so you're not running training but only evaluation. in my case, i am running training so i'm not sure how to help you increase your mAP.
> 
> about the incomplete annotations, we're on the same page haha i'm running a bunch of different tests trying to figure that one out.

Oh... I made a mistake. The training code is 'python train.py --config=yolact_base_config --batch_size=32'.
Then I trained 40000 times on my dataset for 2 days and got the model file 'yolact_base_869_40000.pth'.
I trained for 2 days, but my mAP is around 10.

**Comment by amaanda** [comment]
did you properly adapt your config file as well, with your dataset?
